.github/project.yml

```diff
@@ -1,4 +1,3 @@
 release:
-  current-version: "schmalnau"
-  next-version: "lütter-SNAPSHOT"
-
+  current-version: "0.0.1"
+  next-version: "999-SNAPSHOT"
```

Comment:
Don't change this in this PR otherwise it will trigger a release

Reply:
Reverted!